### PR TITLE
Updated documentation. Fixes #28

### LINKED ISSUE #28: Update documentation.
As described, time to explain how to use the client...

### PR DESCRIPTION
**Pull request by bschroeter**
Latest documentation update. More will come after conda publishing implemented (i.e. installation instructions).

### CODE REVIEW

**Comment by bschroeter** [comment]
Initial documentation updates.

File: docs/index.md
Comment (SeanBryan51) [inline docs/index.md:1]: If you are using mkdocs, you can render markdown from the README.md file using "snippets". I was able to do this for benchcab (see [here](https://github.com/CABLE-LSM/benchcab/commit/9a8ec67ecfd351d1d3d55e2b183886eab922d0a9#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2)).

**Review by SeanBryan51 — APPROVED** [review]
I've left a comment on rendering markdown from the README file but overall I see no issues. Approving.

File: docs/index.md
Comment (bschroeter) [reply docs/index.md:1]: Thanks @SeanBryan51, I'll add that to the next lot of documentation updates that I do.

**Pull request merged**